src/Rel8/Expr/Aggregate.hs

```diff
@@ -46,9 +47,9 @@ import Rel8.Type.Sum ( DBSum )
 
 
 -- | Count the occurances of a single column. Corresponds to @COUNT(a)@
-count :: Expr a -> Aggregate (Expr Int64)
+count :: (i -> Expr a) -> Aggregator i (Expr Int64)
```

Comment:
I've opted to have all `Aggregator` producing functions also bundle the `lmap` function at the same time. 99% of the time you're going to want `lmap`, and I think:

```haskell
aggregate f g
  where 
    f = do
      n <- sum x
      m <- count y
      pure (n, m)
```

is clearer than

```haskell
aggregate f g
  where 
    f = do
      n <- lmap x sum
      m <- lmap y count
      pure (n, m)
```

It also saves users a dependency on `profunctors`.